docs/manuals/source/TasksAndConcepts/CatalogMaintenance.rst

```diff
@@ -291,148 +291,87 @@ If **dbconfig-common** did not succeed or you choose not to use it, run the Bare
-
-   The retention period is specified in seconds, but as a convenience, there are a number of modifiers that permit easy specification in terms of minutes, hours, days, weeks, months, quarters, or years. See :config:datatype:`TIME` for additional details of modifier specification.
-
-   The default Job Retention period is 180 days.
-
-:config:option:`dir/client/AutoPrune`\
-   .. index::
-      single: AutoPrune
-      single: Job; Retention; AutoPrune
-
-   If set to yes, Bareos will automatically apply the File retention period and the Job retention period for the Client at the end of the Job. If you turn this off by setting it to no, your Catalog will grow each time you run a Job.
-
-.. _section-JobStatistics:
-
-Job Statistics
-^^^^^^^^^^^^^^
-
-.. index::
-   single: Statistics
-   single: Job; Statistics
-
-Bareos catalog contains lot of information about your IT infrastructure, how many files, their size, the number of video or music files etc. Using Bareos catalog during the day to get them permit to save resources on your servers.
+So for the above two systems, we should expect to have a database size of approximately 270 Megabytes including the indexes.
+Of course, this will vary according to how many files are actually backed up.
 
-In this chapter, you will find tips and information to measure Bareos efficiency and report statistics.
+You will note that the File table (containing the file attributes) make up the large bulk of the number of records as well as the space used.
 
-If you want to have statistics on your backups to provide some Service Level Agreement indicators, you could use a few SQL queries on the Job table to report how many:
+Without proper setup and maintenance, your Catalog may continue to grow indefinitely read carefully the following sections for planning free space and autovacuuming.
 
--  jobs have run
 
--  jobs have been successful
+.. _FreeSpacePostgres:
 
--  files have been backed up
+Free space needed with PostgreSQL Database
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
--  ...
+To ensure that all PostgreSQL maintenance operations like vacuuming and reindexing roll out smoothly we highly recommend not to fill the disk containing the postgresql data directory to more than 50% during normal operation.
+If this is not possible, you might need to make more space available at least temporary when the database is being upgraded to a new schema version.```
 
-However, these statistics are accurate only if your job retention is greater than your statistics period. Ie, if jobs are purged from the catalog, you won’t be able to use them.
+Normal Bareos operation can create huge temp file requiring free space.
+Upgrading your PostgreSQL version to a new major, will from time to time impose a reindex operation which will use temp space too, especially if option :command:`concurrently` is used.
```

Comment:
```suggestion
Upgrading your PostgreSQL version to a new Bareos major version will, from time to time, impose a reindex operation which will use temp space too, especially if option :command:`concurrently` is used.
```

Reply:
In that case, I was not thinking about major Bareos version (we mostly never do reindex) but obviously new Major PostgreSQL. So I redo the wording to make that more evident.

```suggestion
Upgrading to a new PostgreSQL major version, will from time to time, impose a reindex operation which will use temp space too, especially if option :command:`concurrently` is used.
```